backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5683080_sys_gh14891_OpenItems_Excel_Report.sql

```diff
@@ -0,0 +1,81 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.openitems_excel_report(numeric,
```

Comment:
pls add a file for this function into `ddl` folder too

Reply:
> pls add a file for this function into `ddl` folder too

done